### PR TITLE
Add demo notebook and per-primitive documentation

### PR DESCRIPTION
**Pull request by andyx13**
Added demo notebook and per-primitive documentation

### CODE REVIEW

File: sigpro/sigpro_pipeline_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,485 @@
+{
```
Comment (sarahmish) [inline sigpro/sigpro_pipeline_demo.ipynb:1]: the demo notebook should be separated from the package. I suggest creating a folder under root `tutorials` and move the notebook there.

File: sigpro/sigpro_pipeline_demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+       "2       T001  Sensor1_signal1 2020-01-01 02:00:00   \n",
+       "3       T001  Sensor1_signal1 2020-01-01 03:00:00   \n",
+       "4       T001  Sensor1_signal1 2020-01-01 04:00:00   \n",
+       "\n",
+       "                                             yvalues  sampling_frequency  \n",
+       "0  [0.43616983763682876, -0.17662312586241055, 0....                1000  \n",
+       "1  [0.8023828754411122, -0.14122063493312714, -0....                1000  \n",
+       "2  [-1.3143142430046044, -1.1055740033788437, -0....                1000  \n",
+       "3  [-0.45981995520032104, -0.3255426061995603, -0...                1000  \n",
+       "4  [-0.6380405111460377, -0.11924167777027689, 0....                1000  "
```
Comment (sarahmish) [inline sigpro/sigpro_pipeline_demo.ipynb:305]: based on our previous discussions, is this the new supported format where we have xvalues as a column of type list (like yvalues)? Or is it the original format?

1. If it is the former, I would make `xvalues` a list just like yvalues.
2. If it is the latter, I would change the column name to `timestamp` to follow the dummy data https://raw.githubusercontent.com/sintel-dev/SigPro/master/sigpro/data/demo_timeseries.csv

File: sigpro/sigpro_pipeline_demo.ipynb (hunk 1 above)
Comment (andyx13) [reply sigpro/sigpro_pipeline_demo.ipynb]: Got it - notebook has been moved to the tutorials folder.

File: sigpro/sigpro_pipeline_demo.ipynb (hunk 2 above)
Comment (andyx13) [reply sigpro/sigpro_pipeline_demo.ipynb]: The xvalues column has been revised to be a list, just like yvalues -- should be correct now.

**Review by sarahmish — APPROVED** [review]
LGTM!

**Pull request merged**